scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/DefaultOpEnvironment.java

```diff
@@ -42,11 +53,28 @@
 import org.scijava.log2.Logger;
 import org.scijava.log2.StderrLoggerFactory;
 import org.scijava.meta.Versions;
-import org.scijava.ops.api.*;
-import org.scijava.ops.api.features.*;
+import org.scijava.ops.api.Hints;
+import org.scijava.ops.api.InfoChain;
+import org.scijava.ops.api.InfoChainGenerator;
+import org.scijava.ops.api.OpCandidate;
+import org.scijava.ops.api.OpDependencyMember;
+import org.scijava.ops.api.OpEnvironment;
+import org.scijava.ops.api.OpHistory;
+import org.scijava.ops.api.OpInfo;
+import org.scijava.ops.api.OpInfoGenerator;
+import org.scijava.ops.api.OpInstance;
+import org.scijava.ops.api.OpMetadata;
+import org.scijava.ops.api.OpRef;
+import org.scijava.ops.api.OpWrapper;
+import org.scijava.ops.api.RichOp;
```

Comment:
Hello imports! :wave:

Reply:
Yeah, there are unfortunately a lot of wildcard imports (`git grep import.*\*`) - maybe we should do a pass through at some point to get rid of the rest of them.

Reply:
Made https://github.com/scijava/scijava/issues/126 to track it.